components/roadmap-grid/grid-row.tsx

```diff
@@ -33,13 +33,12 @@ export function GridRow({
   const viewMode = useViewMode();
   const routerQuery = useRouter().query;
   const [closestDateIdx, setClosestDateIdx] = useState(Math.round(globalTimeScaler.getColumn(dayjs.utc(milestone.due_date.get()).toDate())));
-  useEffect(() => {
+  useHookstateEffect(() => {
```

Comment:
note: useHookstateEffect is supposed to recognize hookstate objects, but doesn't seem to work properly, unlike useHookstateMemo (which does work properly, not rerunning the hook callback if hookstate object wasn't changed)